fatiando/vis/mpl.py

```diff
@@ -992,3 +994,93 @@ def pcolor(x, y, v, shape, interp=False, extrapolate=False, cmap=pyplot.cm.jet,
         plot = basemap.pcolor(lon, lat, V, cmap=cmap, vmin=vmin, vmax=vmax,
                               picker=True)
     return plot
+
+
+def seismic_wiggle(section, dt=0.004, ranges=None, scale=1.,
+                   color='k', normalize=False):
+    """
+    Plot a seismic section (numpy 2D array matrix) as wiggles.
+    Slow for more than 200 traces, in this case use `seismic_image`.
```

Comment:
Add a  line between these two sentences and make the second line a warning with:

    Plot a seismic section (numpy 2D array matrix) as wiggles.

    .. warning::        
        Slow for more than 200 traces, in this case use ``seismic_image``.
   
Notice that the format for code in rst is `` (not ` in Markdown). I do this all the time.